include/mega/types.h

```diff
@@ -820,7 +821,7 @@ class CacheableStatus : public Cacheable
     bool serialize(string* data) override;
 
     // deserializes the string to a SyncConfig object. Returns null in case of failure
```

Comment:
```suggestion
    // deserializes the string to a SyncConfig object. Returns null in case of failure.
    // returns a pointer to the unserialized value, owned by client
```

Reply:
It's the same for other `unserialize()` methods, but doc never hurts :) I'll add it.